assets/src/dashboard/components/cardGallery/stories/index.js

```diff
@@ -18,32 +18,23 @@
  * Internal dependencies
  */
 import CardGallery from '../index';
+import formattedTemplatesArray from '../../../storybookUtils/formattedTemplatesArray';
+import FontProvider from '../../../app/font/fontProvider';
+import { TransformProvider } from '../../../../edit-story/components/transform';
 
 export default {
   title: 'Dashboard/Components/CardGallery',
   component: CardGallery,
 };
 
 export const _default = () => {
-  const cards = [
-    'red',
-    'orange',
-    'green',
-    'blue',
-    'purple',
-    'aqua',
-    'yellow',
-    'grey',
-  ].map((color) => (
-    <div
-      key={color}
-      style={{ backgroundColor: color, width: '100%', height: '100%' }}
-    />
-  ));
-
   return (
-    <div style={{ padding: '20px' }}>
-      <CardGallery>{cards}</CardGallery>
-    </div>
+    <FontProvider>
+      <TransformProvider>
+        <div style={{ padding: '20px' }}>
```

Comment:
Let's turn this into a styled component so the inline style isn't generated each render